### ISSUE TITLE
sqlglot fails to parse bigquery partition expression with function call

### ISSUE THREAD

**Issue by jli** [tobymao/sqlglot#512]
example query:
```bigquery
with table as (
  select 1.1 as x
  union all
  select 1.2
  union all
  select 2.3
)
select x, row_number() over (partition by bqutil.fn.int(x))
from table
```

<img width="522" alt="image" src="https://user-images.githubusercontent.com/133466/193232169-88d945d8-c84a-4337-ae10-09b3f096bd6b.png">

sqlglot error:
`error formatting query: ParseError('Expecting ). Line 8, Col: 48.\n  \n  union all\n  select 1.2\n  union all\n  select 2.3\n)\nselect x, row_number() over (partition by f.int\x1b[4m(\x1b[0mx))\nfrom table\n\n\n\nInvalid expression / Unexpected token. Line 8, Col: 51.\n  union all\n  select 1.2\n  union all\n  select 2.3\n)\nselect x, row_number() over (partition by f.int(x)\x1b[4m)\x1b[0m\nfrom table\n\n')`

**Comment by GeorgeSittas** [comment]
Your example query can be parsed by sqlglot just fine in the main branch. Observe:

```python
>>> sql = """
... with table as (
...   select 1.1 as x
...   union all
...   select 1.2
...   union all
...   select 2.3
... )
... select x, row_number() over (partition by bqutil.fn.int(x))
... from table
... """
>>> sqlglot.parse_one(sql).sql()
'WITH table AS (SELECT 1.1 AS x UNION ALL SELECT 1.2 UNION ALL SELECT 2.3) SELECT x, ROW_NUMBER() OVER (PARTITION BY bqutil.fn.INT(x)) FROM table'
```

**Comment by jli** [comment]
ah, I got because I saw @tobymao fixed #509, and v6.2.6 was released. I pulled v6.2.6 and saw #509 was fixed, but this change hasn't been deployed, I guess?

**Comment by tobymao** [comment]
3e93a22 isn't deployed yet

**Comment by jli** [comment]
Ah ok, all clear now. I was impressed with the issue response latency and how frequently sqlglot is released, so I assumed it was included in v6.2.6.

Just wondering when the next release is planned? (I'm waiting to merge a PR with the fix)

**Comment by tobymao** [comment]
just pushed it out
